etc/dbus-serialbattery/battery.py

```diff
@@ -352,6 +352,8 @@ def manage_charge_voltage_linear(self) -> None:
                         self.transition_start_time = current_time
                         self.initial_control_voltage = self.control_voltage
                         chargeMode = "Float Transition"
+                        # Assume battery SOC ist 100% at this stage
+                        self.trigger_soc_reset()
```

Comment:
@transistorgit would it maybe be better to use this also for Daly instead of this?

https://github.com/Louisvdw/dbus-serialbattery/blob/a24d6ef458239a1bc70399f774ae71348e48bf9d/etc/dbus-serialbattery/bms/daly.py#L188-L196